week-3/refactor/format-as-12-hours.js

```diff
@@ -4,3 +4,34 @@
 // Store this expression in a variable and reference it twice in the function in the correct place
 
 // Explain why it makes more sense to store this expression in a variable
+//........................................
+
+
+//Answer
+function convertTo12HourFormat(time) {
+    // first we need to get  the number of hours from the time
+    const hours = Number(time.slice(0, 2));
+  
+    // If it's after noon (more than 12 o'clock), we will adjust the time format
+    if (hours > 12) {
+      return `${hours - 12}:00 PM`;
```

Comment:
you are throwing away the minutes of the time passed into your function. take care of that.